bubblewrap.c

```diff
@@ -453,6 +471,19 @@ drop_caps (bool all)
   if (!is_privileged)
     return;
 
+  if (opt_unshare_user)
+    {
+      caps0 |= CAP_TO_MASK (CAP_CHOWN) | CAP_TO_MASK(CAP_SETUID) | CAP_TO_MASK (CAP_SETGID) | CAP_TO_MASK (CAP_FOWNER) | CAP_TO_MASK (CAP_DAC_OVERRIDE);
+    }
```

Comment:
Are you sure about that? Anyway, you still automatically grant the process properties that it wouldn't have if it were run with that capabilities. For instance if the user is "1000" it can now write to a file owned by user 1000 which is not marked with "u+w" permissions.